### PR TITLE
Add script to create package in release

### PR DESCRIPTION
**Pull request by muffins90**
Edit setup.py .
Create bin wrapper.
Add script to create package in release.

### CODE REVIEW

**Comment by csomh** [comment]
Next step should be to factor out some of the common functionality in a koji base class.

**Comment by lubomir** [comment]
One minor thing: you may want to configure your identity in Git.
https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_first_time

**Comment by muffins90** [comment]
OK, I have this change on my git account.

Best wishes,
Jana Cupova

File: releng_sop/koji_create_package_in_release.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    )
+    parser.add_argument(
+        "--commit",
+        action="store_true",
+        help="Program performs a dry-run by default. Enable this option to apply the changes.",
+    )
+    parser.add_argument(
+        "--owner",
+        required=True,
+        help="Owner package",
```
Comment (dmach) [inline releng_sop/koji_create_package_in_release.py]: rephrase to something else, for example "Package owner"

**Comment by dmach** [comment]
I'd like you to squash the 2 commits into 1.
The 2nd commit doesn't come with any value, it only fixes a problem introduced in the 1st commit.
End users would see unnecessary messages in the git log.

Could you update the docs, at least auto-generate argparse docs for the new command?

**Pull request merged**